templates/nextjs/src/components/Layout.tsx

```diff
@@ -9,9 +9,11 @@ import { useActiveWallet } from "@/hooks/useActiveWallet";
 import { useFaucet } from "@/hooks/useFaucet";
 import Head from "next/head";
 import { bn } from "fuels";
+import { useRouter } from "next/router";
 
 export const Layout = ({ children }: { children: React.ReactNode }) => {
   const { faucetWallet } = useFaucet();
+  const router = useRouter();
```

Comment:
Receiving the following error when using this template:

> Error: NextRouter was not mounted. https://nextjs.org/docs/messages/next-router-not-mounted

<img width="1087" alt="image" src="https://github.com/user-attachments/assets/31f115d9-2853-4f06-9c37-8d3ce9667906">